rps.c

```diff
@@ -0,0 +1,104 @@
+//Emma
+//Rock paper scissors
+//I apologize for spelling opponent wrong in every single secion but once I started I just couldn't change them all. So sorry!
+
+#include <stdlib.h>
+
+#include <time.h>
+#include <stdio.h>
+
+int main(){
+
+int rock, paper, scissor;
+
+
+int scorep = 0;
+int scorec = 0;
+
+int i;
+
+rock = 1;
+paper =2;
+scissor = 3;
+int input;
+int oponent;
+
+	for(i=0;i<=4;i++){
+		printf("Enter your choice \n 1 for rock \n 2 for paper \n 3 for scissors\n Choice: ");
+		scanf("%d", &input);
+		srand(time(NULL));
+		int oponent = rand()%4;
+
+	if(input==1){
```

Comment:
you should have used some else if statements instead of a series of if statements - using only if statements is slightly less efficient because you're having the computer check all those if statements every single time.